src/main/java/com/zebrunner/agent/junit5/JUnit5Adapter.java

```diff
@@ -87,7 +91,9 @@ public void registerTestStart(TestIdentifier testIdentifier) {
             } catch (NoSuchMethodException e) {
                 e.printStackTrace();
             }
-            TestStartDescriptor testStartDescriptor = new TestStartDescriptor(uuid, testName, startedAt, klass, method);
+
+            //TODO: is there a way to get argumentsIndex? or should it be <code>null</code> in this context?
+            TestStartDescriptor testStartDescriptor = new TestStartDescriptor(uuid, testName, klass, method, null);
```

Comment:
argumentsIndex is used for parametrized tests. If there is a list of values to be passed as arguments to the test, this property should contain the index in the list
argumentsIndex comes from TestNG agent where there are data providers. this is actually an optional value, so we can omit it for now